src/graphql/mongoose_graphql_response.erl

```diff
@@ -3,10 +3,10 @@
 -export([term_to_json/1, term_to_pretty_json/1]).
 
 term_to_json(Term) ->
-    jiffy:encode(fixup(Term)).
+    iolist_to_binary(jiffy:encode(fixup(Term))).
```

Comment:
o, nice.
I've checked and `iolist_to_binary(Binary)` would not copy Binary, so looks good.